docbook/Developers_Guide/en-US/Events_Reference_Bug.xml

```diff
@@ -114,6 +114,11 @@
 					the data has not yet been persisted.
 				</para>
 
+				<para>
```

Comment:
We should mark the deprecated events as deprecated in the manual and clarify that they will trigger deprecated errors.